run.py

```diff
@@ -193,23 +193,26 @@ def sanitize_router_address(address):
 
 sx1301_conf['antenna_gain'] = float(os.getenv('GW_ANTENNA_GAIN', 0))
 
-
 # Build local_conf
 gateway_conf = {}
 gateway_conf['gateway_ID'] = my_eui
 gateway_conf['contact_email'] = os.getenv('GW_CONTACT_EMAIL', "")
 gateway_conf['description'] = description
 gateway_conf['stat_file'] = 'loragwstat.json'
-gateway_conf['push_timeout_ms'] = int(os.getenv("GW_PUSH_TIMEOUT", 100)) # Default in code is 100
 
+# Add logging Parse GW_LOGGER env var. It is a string, we need a
+# boolean. Enableing this variable will increase the amount of logging
+# on the resin.io console.
 if(os.getenv('GW_LOGGER', "false")=="true"):
   gateway_conf['logger'] = True
-else:
-  gateway_conf['logger'] = False
+  print ("Packet logging enabled")
 
-if(os.getenv('GW_FWD_CRC_ERR', "false")=="true"):
-  #default is False
-  gateway_conf['forward_crc_error'] = True
```

Comment:
It seems like we lost the "GW_PUSH_TIMEOUT" and the "GW_FWD_CRC_ERR" settings during these changes. Was this unintentional?

Reply:
Unintentional. Please check if corrections are visible.